context/transaction.go

```diff
@@ -6,14 +6,18 @@
 
 package context
 
+import (
+	"context"
+)
+
 type EventChannel struct {
 	Message       chan interface{}
 	Event         chan string
 	AmfUe         *AmfUe
 	NasHandler    func(*AmfUe, NasMsg)
 	NgapHandler   func(*AmfUe, NgapMsg)
-	SbiHandler    func(s1, s2 string, msg interface{}) (interface{}, string, interface{}, interface{})
-	ConfigHandler func(s1, s2, s3 string, msg interface{})
+	SbiHandler    func(ctx context.Context, s1, s2 string, msg interface{}) (interface{}, string, interface{}, interface{})
+	ConfigHandler func(ctx context.Context, s1, s2, s3 string, msg interface{})
```

Comment:
If possible, let's replace interface{} with any (alias). If not, we can leave this for a future PR

```suggestion
	SbiHandler    func(ctx context.Context, s1, s2 string, msg any) (any, string, any, any)
	ConfigHandler func(ctx context.Context, s1, s2, s3 string, msg any)
```

Reply:
Sure!